src/components/table/table_header_cell.tsx

```diff
@@ -32,26 +35,47 @@ export function TableHeaderCell<TData extends RowData>(
 function getThProps<TData extends RowData>(
   header: Header<TData, unknown>,
 ): ThProps {
-  const sorted = header.column.getIsSorted();
-  const canSort = header.column.getCanSort();
-  const style: CSSProperties = {
-    position: 'relative',
-    cursor: canSort ? 'pointer' : undefined,
-  };
-  const onClick = canSort ? header.column.getToggleSortingHandler() : undefined;
-  const children = (
-    <div style={{ display: 'flex', flexDirection: 'row', gap: '5px' }}>
-      <div>
-        {flexRender(header.column.columnDef.header, header.getContext())}
+  const { column } = header;
+
+  const sorted = column.getIsSorted();
+  const canSort = column.getCanSort();
+  const sortingIcon = getSortingIcon(column.columnDef.sortingFn)[
+    sorted || 'asc'
+  ];
+
+  return {
+    onClick: canSort ? column.getToggleSortingHandler() : undefined,
+    style: {
+      position: 'relative',
+      cursor: canSort ? 'pointer' : undefined,
+    },
+    children: (
+      <div style={{ display: 'flex', flexDirection: 'row', gap: '5px' }}>
+        <div>{flexRender(column.columnDef.header, header.getContext())}</div>
+        {sorted && <Icon icon={sortingIcon} />}
       </div>
-      {sorted
-        ? {
-            asc: '🔼',
-            desc: '🔽',
-          }[sorted]
-        : null}
-    </div>
-  );
-
-  return { style, children, onClick };
+    ),
+  };
+}
+
+function getSortingIcon<TData extends RowData>(
+  type: SortingFnOption<TData> | undefined,
+): { asc: IconName; desc: IconName } {
+  return match<
+    SortingFnOption<TData> | undefined,
+    { asc: IconName; desc: IconName }
+  >(type)
+    .with('auto', 'basic', 'datetime', () => ({
+      asc: 'sort-asc',
+      desc: 'sort-desc',
+    }))
+    .with('text', 'textCaseSensitive', () => ({
+      asc: 'sort-alphabetical',
+      desc: 'sort-alphabetical-desc',
+    }))
+    .with('alphanumeric', 'alphanumericCaseSensitive', () => ({
```

Comment:
`alpha` means letter. So it's the alphabetic order that is used here and it should not have the numerical icon.